src/Documenter.jl

```diff
@@ -638,18 +651,17 @@ julia> Documenter.generate("MyPackageName")
 ```
 """
 function generate(pkgname::AbstractString; dir=nothing)
-    # TODO:
-    #   - set up deployment to `gh-pages`
```

Comment:
Hmm, no, I guess not completely. There's only two additional steps that need to be done to deploy out of box:

- add a gh-pages branch on github
- run `import Documenter; Documenter.Travis.genkeys("$pkgname")` and follow instructions.

Do you think these should be done programmatically? It seems possible; github has an API. But maybe I'll save that for another PR?